Sources/SwiftDocC/Infrastructure/External Data/OutOfProcessReferenceResolver.swift

```diff
@@ -51,8 +51,13 @@ import SymbolKit
 public class OutOfProcessReferenceResolver: ExternalDocumentationSource, GlobalExternalSymbolResolver {
     private let externalLinkResolvingClient: ExternalLinkResolving
     
+    @available(*, deprecated, renamed: "id", message: "Use 'id' instead. This deprecated API will be removed after 6.2 is released")
```

Comment:
This is `bundleID` not `id`. Looks like we make some mismatch after the Code Review. cc @d-ronnqvist

Reply:
Thanks. I must have missed that in the PR. If I touch this code again in the near future I can update the deprecation message. If not, this deprecated property is scheduled to be removed after the next version (6.2) is released, so the problem will go away on its own.